Dockerfile

```diff
@@ -272,16 +166,16 @@ RUN ./pkg/rpm/build.sh || true &> /dev/null
 WORKDIR /work
 
 COPY ./confgenerator/default-config.yaml /work/cache/etc/google-cloud-ops-agent/config.yaml
-COPY --from=centos8-build-otel /work/cache /work/cache
-COPY --from=centos8-build-fluent-bit /work/cache /work/cache
-COPY --from=centos8-build-systemd /work/cache /work/cache
-COPY --from=centos8-build-diagnostics /work/cache /work/cache
-COPY --from=centos8-build-wrapper /work/cache /work/cache
+COPY --from=rockylinux8-build-otel /work/cache /work/cache
+COPY --from=rockylinux8-build-fluent-bit /work/cache /work/cache
+COPY --from=rockylinux8-build-systemd /work/cache /work/cache
+COPY --from=rockylinux8-build-diagnostics /work/cache /work/cache
+COPY --from=rockylinux8-build-wrapper /work/cache /work/cache
 RUN ./pkg/rpm/build.sh
 
-FROM scratch AS centos8
-COPY --from=centos8-build /tmp/google-cloud-ops-agent.tgz /google-cloud-ops-agent-centos-8.tgz
-COPY --from=centos8-build /google-cloud-ops-agent*.rpm /
+FROM scratch AS rockylinux8
```

Comment:
Careful. The name `centos8` is used by Rapid builds to refer the build, so I would suggest doing this in a staged manner — (1) rename `centos8-*`->`rockylinux8-*` everywhere, but keep `centos8` as an alias for `rockylinux8`, (2) switch the Rapid project to use `rockylinux8`, (3) remove the alias. This is better as a separate set of steps, so I would suggest not folding it into this PR.